packages/jest-cli/src/reporters/base_reporter.js

```diff
@@ -43,4 +45,14 @@ export default class BaseReporter {
   getLastError(): ?Error {
     return this._error;
   }
+
+  getOptions(): $Shape<Options> {
```

Comment:
no need for this - `options` should be passed as the second argument to the constructor

Reply:
see https://github.com/facebook/jest/blob/dc355000db72a6bbf52ce7d28620497873926c94/packages/jest-cli/src/TestScheduler.js#L258-L282